app/src/main/java/org/openlake/sampoorna/presentation/features/self_care/SelfCareAdapter.kt

```diff
@@ -24,11 +24,11 @@ class SelfCareAdapter(var context: Context, var arrayList: ArrayList<SelfCareMod
 
     override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View {
 
-        var view:View = View.inflate(context, R.layout.card_view_item_grid, null)
-        var icons:ImageView = view.findViewById(R.id.grid_item_self_care_icon)
-        var name:TextView = view.findViewById(R.id.grid_item_self_care_title)
+        val view: View = View.inflate(context, R.layout.card_view_item_grid, null)
+        val icons: ImageView = view.findViewById(R.id.grid_item_self_care_icon)
+        val name: TextView = view.findViewById(R.id.grid_item_self_care_title)
 
-        var listItem:SelfCareModel = arrayList.get(position)
+        val listItem: SelfCareModel = arrayList[position]
 
         icons.setImageResource(listItem.icons!!)
```

Comment:
Use of force unwrapping (!!) is not considered a good practice, and its use should be minimal. Instead, we can use the ?: operator to handle the null case. Although it was not seen in your additions, if you ever see any force unwrap, make sure to point out the contributor.

Reply:
Ok will take care of this 👍